pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -389,6 +389,7 @@ func (e *TokensController) ensureReferencedToken(serviceAccount *v1.ServiceAccou
 				v1.ServiceAccountNameKey: serviceAccount.Name,
 				v1.ServiceAccountUIDKey:  string(serviceAccount.UID),
 			},
+			Labels: serviceAccount.Labels,
```

Comment:
likewise, not obvious to me.  Why would the secret get the labels of the service account?